src/components/MultilineText/MultilineText.js

```diff
@@ -59,7 +59,7 @@ class MultilineText extends React.PureComponent {
     const oneLineHeight = getOneLineHeight(id)
     this.oneLineHeight = oneLineHeight
 
-    if (Math.floor(container.offsetHeight / oneLineHeight) > maxLines) {
+    if (Math.ceil(container.offsetHeight / oneLineHeight) > maxLines) {
```

Comment:
Why `ceil`? What's the edge case?

Reply:
`Math.floor(79/40) = 1`
`Math.ceil(79/40) = 2`

were problems with this behavior in news @DmitriVanGuard